wordpress-mobile/WordPress-iOS
# Featured image using media coordinator

This PR changes the code on the PostSettingsViewController to use the new MediaCoordinator instead of using the MediaService directly.

This PR still doesn't solve the issue of making the setting of feature image async ready. Should I do it on this PR or a separated one?

To test:
 - Start the app
 - Create a new post
 - Go to Post Settings
 - Set a feature image
 - See if all works ok
